test/Microsoft.AspNetCore.SignalR.Tests.Utils/PipeReaderExtensions.cs

```diff
@@ -51,6 +51,19 @@ public static class PipeReaderExtensions
             }
         }
 
+
+        public static async Task ReadAsync(this PipeReader pipeReader, int numBytes)
```

Comment:
Rename to `SkipAsync` since it's not returning the values it reads

Reply:
ConsumeAsync?